nx_cugraph/algorithms/drawing/layout.py

```diff
@@ -0,0 +1,93 @@
+# Copyright (c) 2025, NVIDIA CORPORATION.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import cupy as cp
+import networkx as nx
+import pylibcugraph as plc
+
+from nx_cugraph.utils import networkx_algorithm, outbound_attraction_distribution
+
+__all__ = [
+    "forceatlas2_layout",
+]
+
+
+@networkx_algorithm(
+    extra_params=outbound_attraction_distribution,
+    version_added="25.04",
+    _plc="forceatlas2_layout",
+)
+def forceatlas2_layout(
+    G,
+    pos=None,
+    *,
+    max_iter=100,
+    jitter_tolerance=1.0,
+    scaling_ratio=2.0,
+    gravity=1.0,
+    distributed_action=False,
+    strong_gravity=False,
+    node_mass=None,
+    node_size=None,
+    weight=None,
+    dissuade_hubs=False,
+    linlog=False,
+    seed=None,
+    dim=2,
+    store_pos_as=None,
+):
+    """
+    `seed`, `distributed_action`, `weight`, `node_mass`, `node_size` parameter is
+    currently ignored.
+
+    Only `dim=2` is supported.
+    """
```

Comment:
The docstring gets inlined to the nx docs, so I think they usually look better without using lots of newlines.